### DISCUSSION TITLE
3.0 版本无法自定义 head

### DISCUSSION THREAD

**Discussion by Cyber-Yin** [AlistGo/alist#1683] [Q&A] [answered]
如题所示，在设置的 `Customize head` 选项中用 `<link rel="stylesheet" href="xxx">` 引入 css 样式失败。

是用 Docker 安装的，已经清除了缓存，甚至重装过了依旧无效。

**Comment by wengtad** [comment, marked as answer]
需要到主页刷新页面才会生效，如果在管理页面刷新是无效的，不确定这是不是bug。
